_config.yml

```diff
@@ -679,6 +679,14 @@ gitalk:
   # Available values: en, es-ES, fr, ru, zh-CN, zh-TW
   language:
 
+# utteranc
+# Demo: https://utteranc.es/  http://trumandu.github.io/about/
+# theme: github-light,github-dark,github-dark-orange
+utteranc:
+  enable: false
+  repo:  #Github repo such as :TrumanDu/comments
+  pathname:  pathname
```

Comment:
use `issue-term`, `pathname` is one value of this option.

Reply:
Also you need to make a simple explanation for some option like this or leave an useful link.

Reply:
u have got it.add document in [here](https://github.com/theme-next/theme-next.org/pull/55/files)